### ISSUE TITLE
Function name clash - oor and orr

### ISSUE THREAD

**Issue by laurentheirendt** [adhusch/PaCER#34]
Function `orr` is defined in `oor.m`, which is not coherent.

Error from documentation generation:
```
WARNING: [sphinxcontrib-matlabdomain] Unexpected function name: "orr". Expected "oor" in module "src.Functions".
```

@adhusch, what is the correct name?

cc: @Daniel-Tojal

**Comment by adhusch** [comment]
`oor`, optimal oblique resampling.

please check for regression (i.e. if nowhere `orr` is called instead)

**Comment by laurentheirendt** [comment]
Fixed in #36
